tcrudge/handlers.py

```diff
@@ -568,12 +568,11 @@ async def _get_items(self, qs):
                 items, total = await multi(awaitables)
                 # Set total items number
                 pagination['total'] = total
+            elif self.prefetch_queries:
+                items = await self.application.objects.prefetch(qs,
+                                                                *self.prefetch_queries)                
             else:
-                if self.prefetch_queries:
-                    items = await self.application.objects.prefetch(qs,
-                                                                    *self.prefetch_queries)
-                else:
-                    items = await self.application.objects.execute(qs)
+                items = await self.application.objects.execute(qs)
```

Comment:
Function `ApiListHandler._get_items` refactored with the following changes:

- Merge else clause's nested if statement into elif ([merge-else-if-into-elif](https://docs.sourcery.ai/refactorings/merge-else-if-into-elif/))